images/bigquery/Dockerfile

```diff
@@ -14,20 +14,20 @@
 
 # Includes gcloud (for bq and gsutil), python influxdb client (requires pip), python yaml parser, and jq.
 
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 RUN apt-get update && apt-get install -y \
     git \
     jq \
     wget \
-    python \
-    python-yaml \
-    python-pip && \
+    python3 \
+    python3-yaml \
+    python3-pip && \
     rm -rf /var/lib/apt/lists/*
 
-RUN pip install influxdb google-cloud-bigquery==0.24.0
+RUN pip3 install influxdb==5.2.2 google-cloud-bigquery==1.15.0
```

Comment:
```suggestion
RUN pip3 install influxdb==5.2.2 google-cloud-bigquery==1.15.0 ruamel.yaml==0.16
```

Reply:
During the upgrade I preferred `ruamel.yaml` over `PyYAML` for the reasons I listed: https://github.com/kubernetes/test-infra/pull/13383#issue-296273436 although either would work in this case. So, we should either make *these* changes or revert [the usage of `ruamel.yaml`](https://github.com/kubernetes/test-infra/pull/13912/files#diff-c523c30bbfa8fcd3848917f5cb733ecdR33) or there will be runtime errors.

Reply:
oh please yes let's ruamel.yaml instead, done